core/types/pow/diff.go

```diff
@@ -10,6 +10,10 @@ import (
 	"math/big"
 )
 
+// after 45 days c29 will be the main pow graph
```

Comment:
It's the setting of 15s block time and locks 3 days for c24 before unlocking for c29.  since it will update recently, please remember to update the comments when you refactoring the code.